test/run

```diff
@@ -9,7 +9,7 @@ tools/make-bots
 test/common/pixel-tests pull
 tools/webpack-jumpstart --partial || true
 
-TEST_SCENARIO=${TEST_SCENARIO:-verify}
+export TEST_SCENARIO=${TEST_SCENARIO:-verify}
```

Comment:
Why this?

Reply:
Because `run-tests` and test-foo need to be able to see this; it can't just be internal to `test/run`.

Reply:
But in the case that it is set externally, it'll already get wired through, no?  And in the case that it's set internally (ie: the "verify" scenario) then we don't need it to be wired through since the `.get()` call will return `None`.

I guess it's probably more consistent this way, though.

Reply:
Ah, I see what you mean -- right, we could drop it. Committed locally, will be in the the next round of rebasing/push.

Reply:
I actually don't mind to keep it, now that you explained it :)

Reply:
Pushed.